docs/en/status-communities/transfer-your-community-s-ownership.md

```diff
@@ -8,9 +8,36 @@ hide:
 
 # Transfer your community's ownership
 
- :octicons-tools-24: In progress
+:octicons-beaker-24: Beta
 
-!!! note ""
-     We're working on this content.
+You can transfer your Status Community ownership if you can't manage it and want to pass on the responsibility to someone else. To transfer your community's ownership, you need a [backup of the community's private key][back-up-your-community-s-private-key].
+
+When you transfer the ownership, the new owner's computer becomes a community node with full control of the community, including its settings, permissions and membership approvals.
+
+!!! tip
+    If you need help managing your community, consider adding others as admins. For more information, check out [Co-manage your Status Community][co-manage-your-status-community].
+
+## Security problems with the existing ownership model
+
+The current system for transferring ownership of Status Communities has two important security problems:
+
+- The community owner must share the community's private key with the new owner. Manually transferring the private key from one owner to another is an insecure method of handling a private key.
+- There is no way to be certain that the community is fully transferred. After handing it over to the new owner, the previous owner may still have a copy of the private key. This means that an old owner with malicious intentions could potentially regain access to the community at any time.
```

Comment:
How about changing the first sentence to 'The old owner can still regain access to the community'? Otherwise it sounds like there's a bug that can prevent the transfer from working properly.

Reply:
Uhm. I'm not sure about that. I explain "The old onwer can still regain access to the community" you suggest in the second and third sentence. However, the point I want to make is that the new owner (using this insecure model) can't know for sure the community is fully transferred. Whether you consider this a bug or not doesn't change the result, from my point of view.